scout/parse/variant/compound.py

```diff
@@ -1,43 +1,42 @@
 import logging
+from typing import List
 
 from scout.utils.md5 import generate_md5_key
 
 LOG = logging.getLogger(__name__)
 
 
-def parse_compounds(compound_info, case_id, variant_type):
-    """Get a list with compounds objects for this variant.
+def parse_compounds(compound_info: str, case_id: str, variant_type: str) -> List[dict]:
+    """Get a list with compounds objects(dicts) for this variant.
 
-    Arguments:
-        compound_info(str): A Variant dictionary
-        case_id (str): unique family id
-        variant_type(str): 'research' or 'clinical'
+    Scout IDs do not have "chr" prefixed chromosome names, hence we lstrip that from
+    any compound names.
+
+    We need the case id to construct the correct id, as well as the variant type (clinical or research).
 
-    Returns:
-        compounds(list(dict)): A list of compounds
     """
-    # We need the case to construct the correct id
+
     compounds = []
     if compound_info:
         for family_info in compound_info.split(","):
-            splitted_entry = family_info.split(":")
+            split_entry = family_info.split(":")
```

Comment:
Yes! 🥇 I can still remember the voice of the teacher in high school: "split, split, split" 😄